glib/src/gobject/binding_group.rs

```diff
@@ -166,20 +173,30 @@ impl<'a> BindingGroupBuilder<'a> {
             let _ = Box::from_raw(data as *mut (TransformFn, TransformFn, ParamSpec, ParamSpec));
```

Comment:
```suggestion
            let _ = Box::from_raw(data as *mut (TransformFn, TransformFn, String, ParamSpec));
```

Reply:
I think it does, but i have crashes locally on smething else

Reply:
The problem there was that you pass a `*const u8` and not a `String`. And that `*const u8` would even be freed already in various cases.